sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -6051,6 +6051,38 @@ public void CanGenerateSas_WithVersion_False()
             Assert.IsFalse(versionBlob.CanGenerateSasUri);
         }
 
+        [Test]
+        public void CanGenerateSas_Mockable()
+        {
+            // Arrange
+            var constants = new TestConstants(this);
+            var blobEndpoint = new Uri("https://127.0.0.1/" + constants.Sas.Account);
+
+            // Act
+            var blob = new Mock<BlobBaseClient>(blobEndpoint, GetOptions())
+            {
+                CallBase = true
+            };
+            // Assert
+            Assert.IsFalse(blob.Object.CanGenerateSasUri);
```

Comment:
I'm not sure this tests mockability (does this test work if you remove `virtual` from property?).
I'd remove "CallBase" and try to record some behavior, i.e. `blob.Setup(x => x.CanGenerateSasUri).Returns(false)` or so.

Reply:
Done.